demo/src/app/app.routes.ts

```diff
@@ -40,7 +40,3 @@ const routes: RouterConfig = [
   {path: 'components/tooltip', component: NgbdTooltip},
   {path: 'components/typeahead', component: NgbdTypeahead}
 ];
```

Comment:
Would you mind changing the file name to `app.routing.ts` and using the pattern used in [here](https://github.com/IdeaBlade/angular.io/blob/5eef567b46948dd674affda1ac1dab6172caa434/public/docs/_examples/ngmodule/ts/app/app.routing.ts)

Also notice that RouterConfig is deprecated.

Thanks.